pkg/asset/config/appliance_config.go

```diff
@@ -48,8 +48,8 @@ const (
 	PodmanPull = "podman pull %s"
 
 	// Release
-	templateGetVersion = "oc adm release info %s -o template --template '{{.metadata.version}}'"
-	templateGetDigest  = "oc adm release info %s -o template --template '{{.digest}}'"
+	templateGetVersion = "oc adm release info %s -o template --template '{{.metadata.version}}' --insecure=true"
```

Comment:
Just wondering what's the use-case, is it required for tests/CI purposes?

Reply:
For local development purposes where we want the appliance to read the ephemeral release payload created by dev-scripts. This may not be needed, in the additionalImages approach so I will put it on hold.